spec/controllers/topics_controller_spec.rb

```diff
@@ -12,15 +10,15 @@
       it "sets expiry headers for 30 minutes" do
         get :show, params: { topic_slug: "oil-and-gas" }
 
-        assert_equal "max-age=1800, public", response.headers["Cache-Control"]
+        expect(response.headers["Cache-Control"]).to eq("max-age=1800, public")
       end
     end
 
     it "returns a 404 status for GET topic with an invalid sector tag" do
       stub_content_store_does_not_have_item("/topic/oil-and-gas")
       get :show, params: { topic_slug: "oil-and-gas" }
 
-      assert_equal 404, response.status
+      expect(response.status).to eq(404)
```

Comment:
I know this is a conversion (so only do it if you feel like it), but in the transition_landing_page_controller_spec we have this style (which I like):
`expect(response).to have_http_status(:success)` 
so we could switch this out to:
`expect(response).to have_http_status(:not_found)`